### DISCUSSION TITLE
How can we run the play application in debug mode

### DISCUSSION THREAD

**Discussion by devaj04** [Sunbird-Lern/Community#130] [Installation] [answered]
How can we run the play application in debug mode?

URL - https://github.com/Sunbird-Lern/sunbird-lms-service/tree/master
We want to run the application in debug mode to get a better understanding of the code flow.

CC - @AmiableAnil @Jayaprakash8887

**Comment by Jayaprakash8887** [comment, marked as answer]
@devaj04 Add 'Remote JVM Debug' configuration in IDE as shown in screen shot below
![image](https://github.com/Sunbird-Lern/Community/assets/19702246/8e7cea62-decf-4c4e-aceb-89cc4f21f310)

Instead of 'mvn play2:run', execute 'mvnDebug play2:run' command from 'controller' location of the repository in terminal.

Choose newly added 'Remote JVM Debug' configuration and click on 'Debug' button
![image](https://github.com/Sunbird-Lern/Community/assets/19702246/53f1c8a2-4623-4020-9954-1ad95b11f832)

Trigger any API with debug checkpoints set in code .
